bin/thumbnail

```diff
@@ -313,6 +318,17 @@ function thumb_params() {
 		"scale-to-width")
 			THUMB_PARAMS="${QUALITY} -thumbnail ${WIDTH}"
 			;;
+		"scale-only-down")
+			# get height and width
+			origheight=$($IDENTIFY -format "%h\n" "$IN" | head -1)
+			origwidth=$($IDENTIFY -format "%w\n" "$IN" | head -1)
+
+			if [[ "$WIDTH$HEIGHT"X = "X" || $HEIGHT -gt $origheight || $WIDTH -gt $origwidth ]]; then # requested is bigger on either side, so we keep the original dimensions
```

Comment:
`width` and `height` are both optional. When none if them is provided, we keep the original size. When one of them is given, we respect the one. When both are provided, we respect both. I'll fix that in the code, thanks.